elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/_private/ElytronOidcLogger.java

```diff
@@ -56,4 +57,7 @@ public interface ElytronOidcLogger extends BasicLogger {
     @Message(id = 8, value = "The 'disable-trust-manager' attribute has been set to 'true' so no trust manager will be used when communicating with the OpenID provider over HTTPS. This value should always be set to 'false' in a production environment.")
     void disableTrustManagerSetToTrue();
 
+    @Message(id = 9, value = "Oidc attribute '%s' is not supported")
```

Comment:
s/is not supported/is not supported with the current stability level

Reply:
fixed!